extensions/roc-package-module-dev/package.json

```diff
@@ -28,27 +28,27 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "babel-core": "~6.17.0",
-    "babel-preset-latest": "~6.16.0",
-    "chalk": "~1.1.1",
-    "chokidar": "~1.6.0",
-    "fs-readdir-recursive": "1.0.0",
-    "glob": "~7.1.0",
-    "lodash": "~4.16.2",
-    "output-file-sync": "~1.1.2",
-    "path-exists": "~3.0.0",
-    "pretty-ms": "~2.1.0",
+    "babel-core": "^6.24.1",
+    "babel-preset-latest": "^6.24.1",
```

Comment:
Maybe we should do link in webpack and just use explicit presets and plugins: https://github.com/rocjs/roc-package-webpack/blob/master/extensions/roc-package-webpack-dev/src/roc/index.js#L29 ?